### DISCUSSION TITLE
Babel 8 alpha next release

### DISCUSSION THREAD

**Discussion by coderaiser** [babel/babel#15999] [General] [answered]
Could you please point me when will be next release of Babel 8, I see a lot new features in Babel [v7.23](https://babeljs.io/blog/2023/09/22/7.23.0).

**Comment by nicolo-ribaudo** [comment]
I can release one now :)

**Reply by nicolo-ribaudo** [reply]
Our release process is currently being very unstable, but hopefully https://github.com/babel/babel/actions/runs/6297037866 will work.

**Reply by coderaiser** [reply]
It would be amazing :)!

**Reply by nicolo-ribaudo** [reply]
It failed, I'll try to get to it later today.

**Reply by coderaiser** [reply]
I see the comment `/*#__PURE__*/` is missing in a lot of fixtures, is it correct behavior?

![image](https://github.com/babel/babel/assets/1573141/7b1f62ac-a513-41d7-b9e5-d1d9b78cf276)

**Reply by JLHwung** [reply]
That seems like a bug.

**Reply by coderaiser** [reply]
Looks like tests started to fail after `babel-plugin-transform-jsx`:

![image](https://github.com/babel/babel/assets/1573141/3204b161-465b-4860-bbe9-b009109aebc2)

Maybe just to restart the action? Babel 7 tests succeeded, does they differ somehow from Babel 8?

**Reply by JLHwung** [reply]
Yeah, Babel 8 has different test sets. You can run the test with `BABEL_8_BREAKING=1` env, most of them should be fixture related changes.

**Reply by coderaiser** [reply]
Locally failed test are passing:
![image](https://github.com/babel/babel/assets/1573141/34c63d1e-c6e9-467a-bf8d-1bd0af7a44e2)

I think the problem in CI, an re-start can help

![image](https://github.com/babel/babel/assets/1573141/83ceade6-b739-4a74-b8ee-0ab9a242a628)

Same when I run all tests with `BABEL_8_BREAKING=1 make test`:

![image](https://github.com/babel/babel/assets/1573141/96544c80-1db1-46d6-b416-4ff8dc460631)

**Reply by liuxingbaoyu** [reply]
https://github.com/babel/babel/issues/16000
Being tracked here. :)

**Comment by nicolo-ribaudo** [comment]
I re-triggered a release at https://github.com/babel/babel/actions/runs/6314245796

**Reply by nicolo-ribaudo** [reply]
Actually, https://github.com/babel/babel/actions/runs/6314396052

**Reply by coderaiser** [reply]
Thanks a lot :)!

**Reply by nicolo-ribaudo** [reply, marked as answer]
https://github.com/babel/babel/releases/tag/v8.0.0-alpha.3
